### DISCUSSION TITLE
How to see how much disk space is being used by Docker

### DISCUSSION THREAD

**Discussion by LondonAppDev** [abiosoft/colima#568] [Q&A] [answered]
I was wondering if anyone can tell me how (if at all) you can check the disk usage status for allocated disk space?

So, if I run `colima start --disk 20`, how do I know how much of the 20GB allocated is being used?

**Comment by rfay** [comment, marked as answer]
`colima ssh`  and `df -h /usr/local`

(Edited - before I gave an answer that would show the mounted disk, I think this will do)

**Reply by abiosoft** [reply]
@LondonAppDev you can run the command in one go.

```
colima ssh -- df -h /usr/local
```

**Comment by 1951FDG** [comment]
But that doesn't work I believe in my case, it just states the size that colima was initialized in my case, not the actual usage

| Filesystem  | Size | Used | Avail | Use% |
|--------|--------|--------|--------|--------|
| /dev/root | 96G | 2.0G | 94G | 3% |
